From 06fa688f90e899116102eb9a131f018cb5a63ab9 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 22 Nov 2024 11:49:57 +0100
Subject: [PATCH 1363/1695] plat-stm32mp2: Enable EXTI for STM32MP215x boards

This reverts commit d7c807c8b6a7f2fdee7dacbde0842c51cedc3a77.

The EXTI driver is functional, compile it for STM32MP21xx.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git b7c0437a2b6601649d012e799f45733e2d2ee534 plat-stm32mp2: Enable EXTI for STM32MP215x boards]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I212d49833ad9b032c47fb79039feab72bec7a182
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/420307
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 53fdcbb16..4ba38c116 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -155,10 +155,6 @@ endif
 CFG_TEE_CORE_NB_CORE ?= 2
 CFG_STM32MP_OPP_COUNT ?= 3
 
-ifeq ($(CFG_STM32MP21),y)
-CFG_STM32_EXTI ?= n
-endif
-
 CFG_STM32_BSEC3 ?= y
 CFG_STM32_BSEC_WRITE ?= y
 CFG_STM32_CPU_OPP ?= y
-- 
2.47.2

